packages/framework/src/Component/Image/ImageFactory.php

```diff
@@ -43,36 +43,41 @@ public function __construct(
     /**
      * @param string $entityName
      * @param int $entityId
-     * @param string|null $type
+     * @param array $names
      * @param string $temporaryFilename
+     * @param string|null $type
+     * @throws \Shopsys\FrameworkBundle\Component\Image\Processing\Exception\FileIsNotSupportedImageException
      * @return \Shopsys\FrameworkBundle\Component\Image\Image
      */
     public function create(
         string $entityName,
         int $entityId,
+        array $names,
```

Comment:
it could be more descriptive like `namesIndexedByLocale`

Reply:
changed